packages/components/molecules/f-restaurant-card/stories/RestaurantCard.stories.js

```diff
@@ -46,7 +46,14 @@ RestaurantCardComponent.args = {
         cuisines: ['Mexican', 'Burgers', 'Chinese'],
         imageBadges: [{ text: 'Promoted', textColour: badgeColourSchemes.promoted.text, backgroundColour: badgeColourSchemes.promoted.background }, { text: 'StampCards', textColour: badgeColourSchemes.stampcards.text, backgroundColour: badgeColourSchemes.stampcards.background }],
         contentBadges: [{ text: 'BTA Winner' }, { text: 'Michelin Star' }, { text: 'Tried & Tasted' }, { text: 'New Ownership' }, { text: 'Delivered by Menulog' }, { text: 'A very very very super long unrealistic but necessary to test badge that hopefully never happens' }],
-        newBadgeText: 'NEW'
+        newBadgeText: 'NEW',
```

Comment:
MInor thing; I'd use title-case here and then use CSS `text-transform: uppercase` to convert the casing.

Reply:
We'd probably need a class modifier triggered via props in that case - the badge was part of a previous PR of mine so happy to fix forward but keen to hear how you'd handle that flexibly @DamianMullins - just an uppercase prop etc? 'NEW' is the only example currently where badge text is uppercase

Reply:
can fix in next PR

Reply:
I would probably reach out to design on this and see which badges should be which casing (they may have a set of guidelines to follow for this)